docs/components/truncate.md

```diff
@@ -44,22 +44,22 @@ Number of rows to truncate content. Default value is `1`.
 </template>
 ```
 
-### isTextContent
+### textContent
```

Comment:
kind of wonder if we should "better name" this prop if we're changing it anyway. "textContent" kind of seems like I'm supposed to pass it a string?

Reply:
How about just `text`?

Reply:
`truncateText: boolean`? false by default 🤷🏼

Reply:
Renamed